src/core/ext/filters/client_channel/client_channel.cc

```diff
@@ -460,7 +460,9 @@ class ClientChannel::SubchannelWrapper : public SubchannelInterface {
     if (subchannel_node != nullptr) {
       auto it = chand_->subchannel_refcount_map_.find(subchannel_.get());
       if (it == chand_->subchannel_refcount_map_.end()) {
-        chand_->channelz_node_->AddChildSubchannel(subchannel_node->uuid());
+        if (chand_->channelz_node_ != nullptr) {
```

Comment:
If channelz is not enabled, then there's no reason to populate `subchannel_refcount_map_` in the first place.  So I suggest instead moving this check for `chand_->channelz_node_ != nullptr` to wrap all of the code from line 459 to line 470.

Reply:
Done